.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to _WPStarter_
+
+**First**, let us say: Thank you for contributing! :+1:
+
+The outlined steps in this document are just guidelines, not rules, 
+use your best judgment and feel free to propose changes to this document 
+in [a new pull request](https://github.com/wecodemore/wpstarter/compare).
+
+---
+
+## What should I know before getting started?
+
+In general, it's the following rule:
+
+> Try to play nice with later developers or people using this package.
+
+### Submitting changes
+
+1. Please open [an issue](https://github.com/wecodemore/wpstarter/issues/new) 
+   to discuss your changes before filing a pull request.
+2. When you send a pull request, we will love you forever.
+3. Always add changes to **a new branch** named 
+
+        git checkout -b issue-<issue number>-<topic>
+        # Example
+        # issue-54-feature-title
+
+    An exception are extremely minor changes. Those can go directly to `dev`  
+
+4. Always **file PRs against the `dev` branch**. Nothing goes to `master` without 
+   going to `dev` first. In case you did not do that, please just update/rebase 
+   your pull request.
+5. If you include tests, we will love you in all eternity. 
+   **Hint:** We can always use more test coverage.
+6. When you send a pull request, please try to explain your changes. A list of 
+   keywords is enough.
+
+We will label your issue or PR accordingly so that you can filter the list of 
+issues with the least effort possible. We also assign issues to milestones and 
+release minor or major versions when the list of issues exceeds a certain 
+threshold.
+
+### Commit changes
+
+Please, always write a clear log message for your commits. It's always hard 
+to find changes that introduced bugs. Good messages makes it easier to trace 
+things back to their origin.
+
+We really like the 
+[Angular commit message format](https://github.com/angular/angular.js/blob/5d695e5566212d93da0fc1281d5d39ffee0039a3/CONTRIBUTING.md#commit-message-format) 
+a lot. Here's an example:
+
+```
+<type>(<scope>) <subject>, see #<issue number>
```

Comment:
I'm not keen on the issue number being in the commit title - it takes up  valuable space (if limiting to the usual 50 characters), and again, makes the git repo extremely tied to *GitHub*.

Adding the `see #..` in the commit body is fine.

Be explicit about whether you only want `see`, instead of the `fixes` or `closes` keywords.